obsidian-jupyter.py

```diff
@@ -39,7 +40,9 @@
         if request_body['command'] == 'execute':
             cell['source'] = request_body['source']
             try:
-                nb = client.execute(nb)
+                logger.info(f"***************************** start execute****************************** {cell}")
+                nb = _execute(nb)
```

Comment:
The aim of using the client here is to keep the kernel consistent. I'd expect this change to make things slower, because `execute` creates a new client (see https://github.com/jupyter/nbclient/blob/main/nbclient/client.py#L1269).

Reply:
Sorry, Let me tell you reason: I fond that jupyter in vscode can gets result immediately every time but in obsidian it cost about 6 seconds first time. Then I debuged and tested yesterday and fond that it was `nb = client.execute(nb)` costs time. then I fond that replace it with `_execute(nb)` can faster about 3 seconds, so I made a pull request.